src/Test/Slice.test.js

```diff
@@ -0,0 +1,9 @@
+import { countriesReducer } from '../Redux/Countries/countriesSlice';
+
+test('should return the initial state', () => {
+  expect(countriesReducer(undefined, { type: undefined })).toEqual(
+    {
+      countries: [], status: false, error: false,
+    },
```

Comment:
- [ ] Please add more tests for you components as required 

![image](https://user-images.githubusercontent.com/75126481/235230297-9ab160bd-8ebd-4dfc-b0fe-4481a4e452df.png)